src/routes/note-create.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import { Redirect } from 'react-router';
+
+import { NoteCreatePage } from '@folio/stripes/smart-components';
+
+import { entityTypesTranslationKeys } from '../constants';
+
+export default class NoteCreateRoute extends Component {
+  static propTypes = {
+    history: PropTypes.shape({
+      goBack: PropTypes.func.isRequired,
+    }).isRequired,
+    location: ReactRouterPropTypes.location.isRequired,
+  };
+
+  getReferredEntityData() {
+    const {
+      entityName: name,
+      entityType: type,
+      entityId: id,
+    } = this.props.location.state;
+
+    return {
+      name,
+      type,
+      id,
+    };
+  }
+
+  render() {
+    const {
+      history,
+      location,
+    } = this.props;
+
+    return location.state
+      ? (
+        <NoteCreatePage
+          referredEntityData={this.getReferredEntityData()}
+          entityTypesTranslationKeys={entityTypesTranslationKeys}
+          paneHeaderAppIcon="eholdings"
+          domain="eholdings"
```

Comment:
Wondering if domain (eholdings) will be used elsewhere and could be defined as a constant

Reply:
Nice suggestion! I've created constants which contain the names of the domain and app icon  associated with eholdings.